File: problemsKylePade/problemsKylePade_2894/solution.go
package problem2894

import (
	"encoding/json"
	"log"
	"strings"
)

func differenceOfSums(n int, m int) int {
    
}

func Solve(inputJsonValues string) interface{} {
	inputValues := strings.Split(inputJsonValues, "\n")
	var n int
	var m int

	if err := json.Unmarshal([]byte(inputValues[0]), &n); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[1]), &m); err != nil {
		log.Fatal(err)
	}

	return differenceOfSums(n, m)
}

File: problemsKylePade/problemsKylePade_2894/solution.rs
#![allow(non_snake_case)]
use serde_json::{json, Value};

pub struct Solution;

impl Solution {
    pub fn difference_of_sums(n: i32, m: i32) -> i32 {
        
    }
}

#[cfg(feature = "solution_2894")]
pub fn solve(input_string: String) -> Value {
	let input_values: Vec<String> = input_string.split('\n').map(|x| x.to_string()).collect();
	let n: i32 = serde_json::from_str(&input_values[0]).expect("Failed to parse input");
	let m: i32 = serde_json::from_str(&input_values[1]).expect("Failed to parse input");
	json!(Solution::difference_of_sums(n, m))
}
